test/Microsoft.Dnx.Testing.SampleTests/DnuCommonUtilTests.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace Microsoft.Dnx.Testing.SampleTests
+{
+    [Collection(nameof(SampleTestCollection))]
+    public class DnuCommonUtilTests : DnxSdkFunctionalTestBase
```

Comment:
I've added these tests for testing what I'm working on in the testing infrastructure. Some of these will eventually be removed or moved to something like Testing.Framework.Tests to prevent regression in the test infrastructure.